src/Fantomas.Core.Tests/CompilerDirectivesTests.fs

```diff
@@ -3018,3 +3018,146 @@ type internal CompilerStateCache(readAllBytes: string -> byte[], projectOptions:
     class
     end
 """
+
+[<Test>]
+let ``duplicated trivia printing in ExprAppLongIdentAndSingleParenArgNode with both defines, 2822`` () =
```

Comment:
I think in the past I went with the following approach:
- ``main test, number`` => uses `formatSourceString`
- ``main test, define combination`` => uses `formatSourceStringWithDefines`